packages/core/test/new-api/call.ts

```diff
@@ -186,6 +192,228 @@ describe("call", () => {
     assert.equal(callFuture.from, "0x2");
   });
 
+  it("Should be able to pass an AccountRuntimeValue as from option", () => {
```

Comment:
I added a `string` and `AccountRuntimeValue` of this test to every future that takes a `from`.